code/game/atoms.dm

```diff
@@ -25,11 +25,18 @@
 	var/turf/T = get_turf(src)
 	if(!T)
 		return 0
-	if(T.z != ZLEVEL_CENTCOM)
-		return 0
-	if(T.y > 160)
+
+	if(T.z != ZLEVEL_CENTCOM)//if not, don't bother
 		return 0
-	return 1
+
+	//check for centcomm shuttles
+	for(var/centcom_shuttle in list("emergency", "pod1", "pod2", "pod3", "pod4", "ferry"))
+		var/obj/docking_port/mobile/M = SSshuttle.getShuttle(centcom_shuttle)
+		if(src in M.areaInstance)

```

Comment:
Working as intended, but I suppose checking for turfs in area is less costly.
I'll update the check in another PR.